assets/scss/templates/_header.scss

```diff
@@ -123,6 +123,9 @@
 
   &.open {
     display: flex;
+    &.hidden {
```

Comment:
need hidden inside open?

Reply:
To hide menu when you do hover over templates menu for example. Perhaps open class should be named selected, or active.

Reply:
Moved class to atomic class stylesheet.